### ISSUE TITLE
i386: PANIC: unprotected error in call to Lua API (not enough memory)

### ISSUE THREAD

**Issue by rtsisyk** [tarantool/tarantool#1349]
```
./src/tarantool: version 1.6.8-525-ga571ac0
type 'help' for interactive help
tarantool> string.rep('x', 1024 * 1024 * 1024)
PANIC: unprotected error in call to Lua API (not enough memory)
```

i586-linux-gnu
Reproduced on tarantool/build:i386-debian-jessie Docker image

**Comment by rtsisyk** [comment]
Original LuajJIT doesn't crash:
```
./src/luajit 
LuaJIT 2.1.0-beta1 -- Copyright (C) 2005-2015 Mike Pall. http://luajit.org/
JIT: ON SSE2 SSE3 SSE4.1 fold cse dce fwd dse narrow loop abc sink fuse
> string.rep('x', 1024 * 1024 * 1024)
> 
```

**Comment by rtsisyk** [comment]
Discussed today => postpone.
I fixed box/tuple.test.lua to avoid such problem.

**Comment by nabihach** [comment]
One way to fix this is to install torch with Lua 5.2 instead of LuaJIT, as described [http://torch.ch/docs/getting-started.html#_](url)

**Comment by kostja** [comment]
Can't reproduce:
 % ./tarantool 
Tarantool 1.7.6-73-g51d56f8
type 'help' for interactive help
tarantool> string.rep('x', 1024 * 1024 * 1024)
---
- error: not enough memory
...
